src/components/views/messages/MessageActionBar.tsx

```diff
@@ -94,9 +92,6 @@ const OptionsButton: React.FC<IOptionsButtonProps> = ({
             title={_t("Options")}
             onClick={openMenu}
             isExpanded={menuDisplayed}
-            inputRef={ref}
-            onFocus={onFocus}
-            tabIndex={isActive ? 0 : -1}
```

Comment:
These properties aren't needed anymore because `ContextMenuTooltipButton` -> `MenuItem` already provides `RovingTabIndex` behavior.

The one difference here is that it uses it's own `button` ref `useRovingTabIndex(button);` here. Need to investigate whether that matters.

Reply:
Added back the missing `ref` to the button. Just needed to do although we don't have to use the `ref` from `useContextMenu`, https://github.com/matrix-org/matrix-react-sdk/pull/7336#discussion_r768102034

```js
const [menuDisplayed, buttonRef, openMenu, closeMenu] = useContextMenu();
inputRef={buttonRef}
```